robot_scripts/Support/ebs_helper.robot

```diff
@@ -69,6 +69,8 @@ Focus EBS Forms
     [Documentation]  Uses: AutoIt Returns: None
     ...   Focus the EBS forms.
     Win Activate  Oracle Applications - UAT
+    # nFlags=3 to maximise window
+    Win Set State    Oracle Applications - UAT  strText=${EMPTY}  nFlags=3
```

Comment:
Perhaps the 3 can be assigned to a variable with a descriptive name?

Reply:
It could be. Also thinking could have: 
(a) a dictionary to map the window operations to numbers 
or 
(b) a new keyword that gives a more convenient way of calling `Win Set State` 

Will leave as is though. Try to minimise changes at this late stage.